torch/_inductor/kernel/bmm.py

```diff
@@ -99,6 +111,23 @@ def tuned_bmm(mat1, mat2, *, layout=None):
                 layout=layout,
                 **mm_options(config, m, n, k, layout),
             )
+    static_shape, is_nonzero = _is_static_problem([mat1, mat2], layout)
+    if static_shape and is_nonzero and use_cutlass_template(layout, m, n, k):
+        from ..codegen.cuda.gemm_template import CUTLASSGemmTemplate
+
+        out_layout = FixedLayout(
+            device=layout.device,
+            dtype=layout.dtype,
+            size=WrapperCodeGen.statically_known_list_of_ints_or_none(layout.size),
+        )
+        CUTLASSGemmTemplate.add_cutlass_gemm_choices(choices, out_layout, [mat1, mat2])
+    use_aten = use_aten_gemm_kernels()
+    if len(choices) == 0 and not use_aten:
+        log.warning("No choices for GEMM, using ATen backend as fallback")
+        use_aten = True
+
+    if use_aten:
+        choices.extend([aten_bmm.bind((mat1, mat2), layout)])
```

Comment:
```suggestion
        choices.extend((aten_bmm.bind((mat1, mat2), layout)))
```

Reply:
Yep, this is also a bit overkill anyway as it's only a single value to append. I replaced it with a call to append